src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2200,6 +2213,16 @@ Object toStyleConstants(StyleConstants key, View v) {
             return Integer.valueOf(getValue(null, null));
         }
 
+        @Override
+        public int hashCode() {
+            return Float.hashCode(value);
+        }
+
+        @Override
+        public boolean equals(Object val) {
+            return val instanceof CSS.FontSize size && value == size.value;
```

Comment:
What about different units? `font-size: 1em` shouldn't be equal to `font-size: 1ex`, should it?

What about keywords? Currently, declarations `font-size: smaller` and `font-size: larger` are considered equal.